### ISSUE TITLE
Parallelize with workers

### ISSUE THREAD

**Issue by runspired** [dzek69/brotli-cli#15]
Compressing at 11 is hard work, and when you've got a lot of files but also a lot of cores ... profit

**Comment by dzek69** [comment]
Thanks for your suggestion! I finally had some time to address it.

I started with implementing Workers, but I think I went the wrong way - starting a worker is a ~500ms overhead on my pretty fast machine. So doing it once per file was a bad idea.

Probably a better idea would be to split the list of files to compress to equal parts or some more advanced algorithms... but...

We had node v6 AFAIK when `brotli-cli` was first published. Workers are available since v12. And since ~v10 there is a stable zlib built-in module available, which has brotli support and which is truly async, much faster than JS implementation and also bugless (`brotli` npm module can crash on some files since v13-16 (I'm not sure exactly)).

I just released `brotli-cli` 2.0.0, which is API-compatible with the old one, it only requires Node 12+. And it's around 10x faster (it really depends on what you are compressing, it can get much more above that) even with window size parameter being bumped for better but slower compression by default. And by default it will go with concurrency of amount of your CPUs (controllable with new parameter). It seems to be the best setting for maximum performance.

So tl;dr - just bump a version and enjoy!
